components/analytics/team_analytics/index.ts

```diff
@@ -27,7 +27,7 @@ function mapStateToProps(state) {
     };
 }
 
-function mapDispatchToProps(dispatch) {
+function mapDispatchToProps(dispatch: any) {
```

Comment:
Usually this is either `Dispatch` or `Dispatch<GenericAction>`.
Have a look at some other `index.ts` if you need some ideas of what to use here.